src/katsdpcalproc/pointing.py

```diff
@@ -236,23 +233,37 @@ def __call__(self, x):
         """
         return self._interp(x)
 
+# Helper function to calculate expected width
+
+
+def _get_widths(beamwidth, freqs, diameter):
```

Comment:
This function can remain under the radar as an "underscore function" for now, i.e. it's not part of the main user API for the module but an internal function. Because of that documentation is allowed to be a little skimpier. I would consider renaming the arguments though to be more accurate since there are so many "widths" floating around.

How about
```
def _voltage_beamwidths(beamwidth_factor, frequency, dish_diameter):
```
It's typically called one frequency at a time.